HW1/P4/P4.txt

```diff
@@ -1 +1,27 @@
 # Your discussion here
+Basically, this graph has 6444 nodes.
+Using Characters-Comic data, I generated RDD for graph with the form of RDD (node, [a list of neighbors])
+In BFS function, the RDD needs to be updated with distances from a root node. So, I changed the form to RDD (node, (distance, [a list of neighbors]))
+
+Here are the results for the three cases
+1) Root Node : CAPTAIN AMERICA
+The number of touched nodes : 6407
+The number of untouched nodes : 37
+Diameter : 3
+
+2) Root Node : MISS THING/MARY 
+The number of touched nodes : 6
+The number of untouched nodes : 6438
+Diameter : 1
+
+3) Root Node : ORWELL
+The number of touched nodes : 8
+The number of untouched nodes : 6436
+Diameter : 1
+
```

Comment:
I got the same thing for the final numbers. Only I accounted for the character itself (Node0).